release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,23 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 09 September 2024 - Enhancement - Catalog - Catalog item versions with no artifact available in the cloud will contain a message [ID_40616]
+
+When a Catalog item version is not available to deploy because its installation file is not registered, a message will be shown to the user as not being available and a request to contact support.cloud.ecosystem@skyline.be
+
+#### 09 September 2024 - New Feature - Catalog API - Catalog Registration supports images in description readme file [ID_40219]
```

Comment:
Do we want to specify this is for private items? since it was already doable for public items.

Also don't know if we need to mention that images hosted elsewhere also work.
(^^ Feels this is more a documentation thing than a changelog thing)

Reply:
as discussed was ok to leave in